dev-libs/nss/nss-3.56.ebuild

```diff
@@ -163,6 +163,12 @@ multilib_src_compile() {
 	export ASFLAGS=""
 	# Fix build failure on arm64
 	export NS_USE_GCC=1
+	# Detect compiler type and set proper environment value
+	if tc-is-gcc; then
+		export CC_IS_GCC=1
```

Comment:
Now shouldn't you move `export NS_USE_GCC=1` into "tc-is-gcc" clause?

Reply:
This NS_USE_GCC variable is also needed by clang to compile successfully in arm64. As I said earlier, this flag is only to indicate that the compiler supports GCC's C extension, which is also supported by clang.